ros/lib/utils.py

```diff
@@ -151,11 +151,8 @@ def insert_performance_profiles(session, system_id, fields):
        performance_profile_history table.
     """
     fields = {} if fields is None else fields
-    old_profile_record = session.query(PerformanceProfile).filter_by(
-        system_id=system_id).first()
-    if old_profile_record:
-        session.delete(old_profile_record)
-        session.commit()
+    session.execute(db.delete(PerformanceProfile).filter(PerformanceProfile.system_id == system_id))
```

Comment:
I think here we can just delete the PerformanceProfile instead of first checking if it exists? @r14chandra @kgaikwad ?

Reply:
Yeah,  we can do it. Just one thought, Does `commit`  have any consequences if we don't have anything to `execute`? As I see if we don't have anything to delete, the commit will always be executed. 
If not, then we can have this change.

Reply:
As per my understanding running commit without deleting anything is not harmful. specially below para from the sqlalchemy(https://docs.sqlalchemy.org/en/20/orm/session_basics.html#committing),

> When there is no transaction in place for the [Session](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session), indicating that no operations were invoked on this [Session](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session) since the previous call to [Session.commit()](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.commit), the method will begin and commit an internal-only “logical” transaction, that does not normally affect the database unless pending flush changes were detected, but will still invoke event handlers and object expiration rules.